docs/hw/mechanical.md

```diff
@@ -41,6 +43,53 @@ The typical toy brick has its studs spaced at 8 mm, while Create® 3's holes are
 It is possible to place an [adapter (576 kB)](data/C3-Stud-Mount.stl) into every other hole in order to connect toy bricks to the Create® 3.
 We recommend printing at 100% infill.
 
+## Adding a rear caster
+Out of the box, Create® 3  has a center of gravity in front of the two drive wheels and features an integrated front caster to act as a third point of contact.
+If you add a payload that is heavy and/or causes the robot center of gravity to move behind the drive wheels, you may want to add a second caster wheel in the rear to prevent sliding along the back edge.
+Below you will find designs for two, 3D-printable solutions which add a second [Roomba® caster wheel](https://store.irobot.com/default/parts-and-accessories/roomba-accessories/700-series/roomba-front-caster-wheel/4624869.html) to the rear of the Create® 3.
+
+### Rear caster attachment
+The first option is a smaller print (99 x 83 x 58 mm) that attaches to the rear of the Create® 3 cargo bay.
+It is faster and simpler to build but protrudes 54 mm behind the rear of the robot, so it may not be suitable for situations were the robot needs to navigate tight spaces or in cluttered environments.
+
+![Rear caster attachment](data/caster_attachment.png "Rear caster attachment")
+
+The rear caster attachment is held in place between the handle and cable passthrough of the cargo bay using a printed latch and two M3 or #4 self-tapping screws.
+The Roomba® caster wheel is inserted into the bottom opening and held in place with a small locking piece which is then screwed into place.
+If the caster does not rotate freely, sand or file the bearing surface along the caster's axis of rotation.
+
+| Part | Quantity |
+| --- | --- |
+| [Roomba® caster wheel](https://store.irobot.com/default/parts-and-accessories/roomba-accessories/700-series/roomba-front-caster-wheel/4624869.html) | 1 |
+| [Rear caster attachment](data/C3-Rear-Caster-Attachment.stl) | 1 |
+| [Caster lock](data/C3-Caster-Lock.stl) | 1 |
+| [Top latch](data/C3-Rear-Caster-Attachment-Latch.stl) | 1 |
+| M3 or #4 self-tapping screw | 3 |
+
+### Cargo bay with integrated caster
+The second option replaces the included Cargo Bay with one that contains an integrated rear caster.
+It requires a larger print volume (208 x 201 x 66 mm) and slightly reduces the cargo space, but the solution does not protrude beyond the back of the robot.
+This allows the robot to turn in tight spaces and reduces the likelihood of the caster attachment getting caught on objects in the environment.
+
+![Cargo bay with caster](data/cargo_bay_caster.png "Cargo bay with caster")
```

Comment:
Love this.

Reply:
FWIW, it fit and printed well on my Ender 3 S1! (I had to disable the default skirt to make Cura realize it would fit though)